app/src/main/java/com/mithridat/nonoconverter/ui/result/StringKeys.java

```diff
@@ -9,4 +9,6 @@
     static final String TOTAL_SCALE = "Total scale";
     static final String CENTER_RELATIVE_X = "Relative center X";
     static final String CENTER_RELATIVE_Y = "Relative center Y";
+    static final String DIALOG_HOME_RETURN_TAG = "Fragment home-return dialog";
+    static final String DIALOG_DONE_TAG = "Fragment done dialog";
```

Comment:
Where are the comments?

Reply:
There were no comments for all other fields, I thought it's quite unnecessary.